x/ibc/transfer/keeper/migrations.go

```diff
@@ -0,0 +1,27 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+
+	v3 "github.com/evmos/evmos/v10/x/ibc/transfer/migrations/v3"
+)
+
+var _ module.MigrationHandler = Migrator{}.Migrate2to3
+
+// Migrator is a struct for handling in-place store migrations.
+type Migrator struct {
+	keeper Keeper
+}
+
+// NewMigrator returns a new Migrator.
+func NewMigrator(keeper Keeper) Migrator {
+	return Migrator{
+		keeper: keeper,
+	}
+}
+
+// Migrate2to3 migrates from consensus version 1 to 2.
```

Comment:
As I understand consensus versioning in upgrades logic, this should be "2 to 3"
```suggestion
// Migrate2to3 migrates from consensus version 2 to 3.
```